pybind_interface/cuda/pybind_main_cuda.cpp

```diff
@@ -27,8 +27,10 @@ namespace qsim {
       unsigned num_sim_threads,
       unsigned num_state_threads,
       unsigned num_dblocks
-    ) : ss_params{num_state_threads, num_dblocks} {}
-
+    ) {
```

Comment:
What motivates this change? AFAICT the two should behave identically.

Reply:
Locally on my Linux box, as well as on the conda-forge building infrastructure the build will fail with:
```
./pybind_interface/cuda/pybind_main_cuda.cpp(30): error: no instance of constructor "qsim::StateSpaceCUDA<FP>::Parameter::Parameter [with FP=float]" matches the argument list
```